core/global_include.cpp

```diff
@@ -32,6 +32,11 @@ double Time::elapsed_since_s(const dl_time_t &since)
            static_cast<double>(steady_clock::period::den);
 }
 
+double Time::elapsed_since_ms(const dl_time_t &since)
```

Comment:
I don't think we should supply this function. You only us it once in the code you added, and it is weird to have a `double` for milliseconds.

Reply:
ok, I'll remove it.